about.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>About Web Development Bootcamp</title>
+  </head>
+  <body>
+    <section class="main-section main-section-about">
+      <nav class="mobile-menu">
+        <img src="images/enabled.jpg" alt="Enable icon" class="enabled-menu" />
```

Comment:
Remove this partners section from about page since there isn't one on the template.

![Screenshot from 2023-08-02 15-16-15](https://github.com/memicelis/web-development-bootcamp/assets/118063058/34b39111-3ec2-441c-adb0-0b42e954e9e8)

![Screenshot from 2023-08-02 15-16-07](https://github.com/memicelis/web-development-bootcamp/assets/118063058/03d5b975-ba4e-4c11-841c-807e48e46f24)